.github/workflows/python-app.yml

```diff
@@ -68,7 +69,9 @@ jobs:
           python-version: 3.8
       - name: Install dependencies
         run: |
+          pip install -r dev-requirements.txt
```

Comment:
I can take a closer look and might ask you some follow up questions. In essence, I was seeing build errors because the `pybind11` requirement for the catanatron_compiled wasn't installed. But I can try to revisit this.

Reply:
Prefacing with `dev-requirements.txt` ensured that the dev requirements were installed including `pybind11` but yea that's probably too broad.